book/src/pong-tutorial/pong-tutorial-03.md

```diff
@@ -156,15 +164,51 @@ impl<'s> System<'s> for PaddleSystem {
 # fn main() {}
 ```
 
-Now lets add this system to our `GameDataBuilder` in `main.rs`:
+Now, there's quite a bit going on here!
+
+We create a unit struct `PaddleSystem`, and implement the `System` trait for it
+with the lifetime of the components on which it operates.
+Inside the implementation, we define the data the system operates on in the
+`SystemData` tuple: `WriteStorage`, `ReadStorage`, and `Read`. More
+specifically, the generic types we've used here tell us that the `PaddleSystem`
+mutates `Transform` components, `WriteStorage<'s, Transform>`, it
+reads `Paddle` components, `ReadStorage<'s, Paddle>`, and also accesses the
```

Comment:
I know we use `'a` in most places instead of `'s` so I'm curious how this might impact confusion.

Reply:
Note that this was in the text before, I just moved it. I didn't find the use for `'s` confusing, personally.